Physics/Dynamics/ProductionFormFactor.hpp

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2013, 2019 The ComPWA Team.
+// This file is part of the ComPWA framework, check
+// https://github.com/ComPWA/ComPWA/license.txt for details.
+
+#ifndef PHYSICS_DYNAMICS_PRODUCTION_FORMFACTOR_HPP_
+#define PHYSICS_DYNAMICS_PRODUCTION_FORMFACTOR_HPP_
+
+#include <boost/property_tree/ptree.hpp>
+#include <memory>
+#include <vector>
+
+#include "AbstractDynamicalFunction.hpp"
+#include "Core/Exceptions.hpp"
+#include "Core/Functions.hpp"
+#include "Core/Spin.hpp"
+#include "FormFactor.hpp"
+
+namespace ComPWA {
+namespace Physics {
+namespace Dynamics {
+
+/// The production formfactor is implemented in Blatt-Weisskopf type \f$B_{L}(q)\f$
+/// Ref. Dalitz Plot Analysis, PDG 2012, where 
+/// \f[
+/// \frac{B^{\prime}(q_{0})}{B^{\prime}_{L}(q)} = q^{L}B^{\prime}_{L}(q, q_{0})
+/// \f]
+class ProductionFormFactor
+    : public ComPWA::Physics::Dynamics::AbstractDynamicalFunction {
+
+public:
+  //============ CONSTRUCTION ==================
+  ProductionFormFactor(std::string name,
+                          std::pair<std::string, std::string> daughters,
+                          std::shared_ptr<ComPWA::PartList> partL);
+  virtual ~ProductionFormFactor();
+
+  boost::property_tree::ptree save() const;
```

Comment:
This function was removed. Don't implement it

Reply:
Yes, I just removed it.